modules/programs/rofi/default.nix

```diff
@@ -0,0 +1,90 @@
+{ mode, config, pkgs, lib, ... }:
+
+with lib;
+let
+  i3Package =
+    if mode == "home-manager" then config.xsession.windowManager.i3.package
+    else config.services.xserver.windowManager.i3.package;
+
+  i3Support = pkgs.stdenvNoCC.mkDerivation rec {
```

Comment:
This should be moved to `pkgs`. Also check how can it be integrated as a [rofi plugin](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/misc/rofi/wrapper.nix).